var/spack/repos/builtin/packages/mrbayes/package.py

```diff
@@ -32,9 +32,10 @@ class Mrbayes(AutotoolsPackage):
        of model parameters."""
 
     homepage = "http://mrbayes.sourceforge.net"
-    url      = "https://downloads.sourceforge.net/project/mrbayes/mrbayes/3.2.6/mrbayes-3.2.6.tar.gz"
+    url      = "https://github.com/NBISweden/MrBayes"
 
-    version('3.2.6', '95f9822f24be47b976bf87540b55d1fe')
+    version('3.2.6', commit='8a9adb11bcc538cb95d91d57568dff383f924503',
```

Comment:
on their release page it's `425e8ce` https://github.com/NBISweden/MrBayes/releases

Reply:
and there is also a tarbal, so it's probably better to use it

Reply:
oops, forgot to change the version. But you think I should use the "pre-release" rather than the most recent commit? I typically take the most recent commit in my other PRs.

Reply:
There's been 82 commits to mrbayes/develop since that release

Reply:
> But you think I should use the "pre-release" rather than the most recent commit?

well, this is a stable reelase `3.2.6`, so I think you should use the tarball. Whereas for `develop` just checkout the HEAD of the corresponding branch.

Reply:
Yes, but the reason I am putting in this pull request is to fix #6564, and in the 3.2.6 release the problem is still there